joblib/test/test_memory.py

```diff
@@ -514,6 +514,41 @@ def test_call_and_shelve_argument_hash(tmpdir):
         in str(w[-1].message)
 
 
+def test_call_and_shelve_performance(tmpdir):
+    """Check call_and_shelve only load stored data if needed."""
+    test_access_time_file = tmpdir.join('test_access')
+    test_access_time_file.write('test_access')
+    test_access_time = os.stat(test_access_time_file.strpath).st_atime
+    # check file system access time stats resolution is lower than test wait
+    # timings.
+    time.sleep(0.5)
+    assert test_access_time_file.read() == 'test_access'
+
+    if test_access_time == os.stat(test_access_time_file.strpath).st_atime:
+        # Skip this test when access time cannot be retrieved with enough
+        # precision from the file system (e.g. NTFS on windows).
+        return
```

Comment:
Please use `pytest.skip("filesystem does not support fine-grained access time attribute")` to make it explicitly that the test is skipped.